src/style.css

```
@@ -1 +1,97 @@
-/* add styles here */
+@font-face {
+  font-family: Avenir;
+  src: url(fonts/Avenir-Book.ttf);
+  font-weight: 300;
+}
+
+@font-face {
+  font-family: AvenirFont;
+  src: url(fonts/Avenir-Heavy.ttf);
+  font-weight: 300;
+}
+
+html {
+  font-family: AvenirFont, sans-serif;
+}
+
+.container {
+  padding-top: 12px;
+  color: #3d4e61;
+}
+
+.search-bar {
+  position: relative;
+  width: 100%;
+}
+
+.input {
+  color: #3d4e61;
+  font-weight: 300;
+  width: 100%;
+  border-radius: 4px;
+  box-sizing: border-box;
+  border: 1px solid #e1e7ed;
+  box-shadow: 0 1px 8px 0 #3d4e611a;
+  gap: 0;
+}
+
+.input__image {
+  position: absolute;
+  top: 50%;
+  transform: translateY(-50%);
+}
+
+.input__big {
+  height: 70px;
+  margin-bottom: 20px;
+  font-family: Avenir, sans-serif;
+  font-size: 16px;
+}
+
+.input__big .input__image {
```

```suggestion
.input__image--big {
```
redundant combined selector. Add specific class for big version of image (same as for input and form)